frontend/src/layout/navigation/Navigation.stories.tsx

```diff
@@ -18,74 +14,57 @@ const meta: Meta = {
-                            {
-                                name: 'Torpurator',
-                                function: 'Spontaneously combusts',
-                                color: 'Chartreuse',
-                                ionizationLevel: 100,
-                            },
-                            {
-                                name: 'De-Blargifier',
-                                function: 'De-radicalizes blue whales',
-                                color: 'Beige',
-                                ionizationLevel: -423,
-                            },
-                        ]}
-                    />
-                </div>
-            </SideBar>
-        </>
+        <SideBar>
+            <div className="main-app-content">
+                <PageHeader
+                    title="Your gizmos"
+                    caption="View your household devices."
+                    buttons={<LemonButton type="primary">New gizmo</LemonButton>}
+                />
+                <LemonTable
+                    columns={[
+                        { title: 'Name', dataIndex: 'name' },
+                        { title: 'Function', dataIndex: 'function' },
+                        { title: 'Color', dataIndex: 'color' },
+                        { title: 'Ionization level', dataIndex: 'ionizationLevel' },
+                    ]}
+                    dataSource={[
+                        // Nonsensical data for demo purposes
+                        {
+                            name: 'Blargifier',
+                            function: 'Radicalizes blue whales',
+                            color: 'Azure',
+                            ionizationLevel: 423,
+                        },
+                        {
+                            name: 'Frink',
+                            function: 'Makes the world go round',
+                            color: 'Crimson',
+                            ionizationLevel: 0,
+                        },
+                        {
+                            name: 'Torpurator',
+                            function: 'Spontaneously combusts',
+                            color: 'Chartreuse',
+                            ionizationLevel: 100,
+                        },
+                        {
+                            name: 'De-Blargifier',
+                            function: 'De-radicalizes blue whales',
+                            color: 'Beige',
+                            ionizationLevel: -423,
+                        },
+                    ]}
+                />
+            </div>
+        </SideBar>
     )
 }
 
 export function AppPageWithSideBarHidden(): JSX.Element {
-    const { toggleSideBarBase, toggleSideBarMobile } = useActions(navigationLogic)
-
-    useEffect(() => {
-        toggleSideBarBase(false)
-        toggleSideBarMobile(false)
-    }, [])
-
     return <BaseAppPage />
 }
 
 export function AppPageWithSideBarShown(): JSX.Element {
```

Comment:
This story and the one above look to do the same thing. Should we consolidate or remove them if no longer necessary?